pkgs/top-level/aliases.nix

```diff
@@ -365,7 +365,6 @@ mapAliases ({
   speedtest_cli = speedtest-cli;  # added 2015-02-17
   spice_gtk = spice-gtk; # added 2018-02-25
   spice_protocol = spice-protocol; # added 2018-02-25
-  spidermonkey_52 = throw "spidermonkey_52 has been removed. Please use spidermonkey_60 instead."; # added 2019-10-16
```

Comment:
Firefox 52 has not been receiving security updates for over a year. Because this is security critical component, its re-introduction is not a good idea. Could you ask upstream to switch to a newer Spidermonkey version?

Reply:
https://github.com/NixOS/nixpkgs/pull/74242#discussion_r350716404

Reply:
55 isn't an ESR, afaik? Wouldn't that be worse, then?

Reply:
I've done some research and it doesn't seem any distro is currently using a newer spidermonkey version. I think we should ask upstream.

Reply:
https://github.com/linuxmint/cjs/issues/80

Reply:
Upstream https://github.com/linuxmint/cjs/issues/80

Should we continue with adding spidermonkey 52 if it doesn't get resolved in time?

Reply:
Upstream said 4.4.0: wontfix, 4.6.0: likely, 4.8.0: definitively.
Since 4.4.0 came out basically this week, we can assume that we'll either have to wait until 4.6.0 or add 52 for 4.4.0 one last time.

Reply:
Sounds good to me.